tests/state_test_util.go

```diff
@@ -251,6 +261,15 @@ func (t *StateTest) RunNoVerify(subtest StateSubtest, vmconfig vm.Config, isTest
 				DeferredTxFee: true,
 			},
 		}
+		if config.IsOsakaForkEnabled(big.NewInt(int64(t.json.Env.Number))) {
+			config.BlobScheduleConfig = &params.BlobScheduleConfig{
+				Osaka: &params.BlobConfig{
+					Target:         6,
+					Max:            9,
+					UpdateFraction: 5007716,
+				},
+			}
+		}
```

Comment:
Is it ok to fix this? Json files contains this as `config`.

Reply:
There doesn't seem to be any tests that are affected at the moment.

Reply:
Let's create a `btConfig` and receive it. That would be more reliable.

Reply:
FIxed at [9ba35f2](https://github.com/kaiachain/kaia/pull/696/commits/9ba35f2177d63aae214af789462cddd0f38b8b28). PTAL